generators/cleanup.js

```diff
@@ -265,6 +265,8 @@ function cleanupOldFiles(generator) {
       generator.removeFile(`${ANGULAR_DIR}core/user/user.model.ts`);
       generator.removeFile(`${ANGULAR_DIR}core/user/user.service.ts`);
       generator.removeFile(`${ANGULAR_DIR}core/user/user.service.spec.ts`);
+      generator.removeFile(`${ANGULAR_DIR}core/config/application-config.service.ts`);
+      generator.removeFile(`${ANGULAR_DIR}core/config/application-config.service.spec.ts`);
```

Comment:
```suggestion
      generator.removeFile(`${ANGULAR_DIR}core/config/config.service.ts`);
      generator.removeFile(`${ANGULAR_DIR}core/config/config.service.spec.ts`);
```
And create generator.isJhipsterVersionLessThan('7.0.0-beta.2')

Reply:
done